c-scape/src/time/mod.rs

```diff
@@ -3,25 +3,81 @@ use core::convert::TryInto;
 use errno::{set_errno, Errno};
 use libc::c_int;
 
+// So 32-bit time support was only added to 64bit in 5.1,
+// so I'm not even sure what Rustix's deal with this is.
+
+// TODO: clarify
```

Comment:
It looks like you've fixed the main problems here. Can you say what the remaining open questions are?